doc/changelog.md

```diff
@@ -91,6 +91,9 @@
 * Add support for generating OpenQasm3 kernels from the ``QuantumDevice`` API in the runtime.
   [#118](https://github.com/PennyLaneAI/catalyst/pull/118)
 
+* Add end-to-end support for execution of OpenQasm3 kernels on Amazon Braket devices.
+  [#139](https://github.com/PennyLaneAI/catalyst/pull/139)

```

Comment:
We should add an example here to showcase this feature :)

Reply:
Added here [c9dad1e](https://github.com/PennyLaneAI/catalyst/pull/139/commits/c9dad1e91039f75f6eca3ff4d897551a3737a4ad)